### PR TITLE
Update view.md with new APIs for issue #929

### PR DESCRIPTION
**Pull request by ndejesus1227**
☂️☂️☂️☂️☂️☂️ #929  ☂️☂️☂️☂️☂️☂️☂️ 

New APIs added:
-nextFocusDown
-nextFocusForward
-nextFocusLeft
-nextFocusUp
-clickable

I don't have much details on any of the `nextFocus` apis, the descriptions in the code are very vague and tell me to refer to the docs which is pretty funny because I'm adding them to the docs.

Is there any easier way to figure out what new stuff has been added? I pretty much found the new APIs in the middle of View.js file, it didn't seem to have any particular order. I was hoping that new Apis would at least be at the very bottom from oldest to newest going from top to bottom.

### CODE REVIEW

File: docs/view.md [hunk 1]
```diff
@@ -81,6 +81,14 @@ For `View` responder props (e.g., `onResponderMove`), the synthetic touch event
 - [`accessibilityElementsHidden`](view.md#accessibilityElementsHidden)
 - [`accessibilityIgnoresInvertColors`](view.md#accessibilityIgnoresInvertColors)
 - [`shouldRasterizeIOS`](view.md#shouldrasterizeios)
+- [`nextFocusDown`](view.md#nextfocusdown)
+- [`nextFocusForward`](view.md#nextfocusForward)
+- [`nextFocusLeft`](view.md#nextfocusleft)
+- [`nextFocusRight`](view.md#nextfocusright)
+- [`nextFocusUp`](view.md#nextfocusleft)
+- [`clickable`](view.md#clickable)
```
Comment (cpojer) [inline docs/view.md]: can you sort this list alphabetically?

**Comment by cpojer** [comment]
Check out https://developer.android.com/training/keyboard-input/navigation – these are part of Android and are related to keyboard navigation.

File: docs/view.md (hunk 1 above)
Comment (ndejesus1227) [reply docs/view.md]: Just to clarify, do you want me to make everything alphabetical? From lines 50 - 89 there seems to be no particular order.

Comment (cpojer) [reply docs/view.md]: Yes, let's get everything alphabetized.

**Comment by ndejesus1227** [comment]
@cpojer how's this for an explanation on the nextFocus stuff:

"designates the next view to receive focus when the user navigates up/down/left/right"?

**Comment by cpojer** [comment]
Works for me!

**Comment by ndejesus1227** [comment]
@cpojer I've made edits based off all of your feedback. Let me know if there's more I need to do

**Pull request merged**